lib/resources/postgres.rb

```diff
@@ -59,7 +55,7 @@ def initialize
         @data_dir = '/var/lib/pgsql/data'
       end
 
-      @service = 'postgresql'
+      @service = 'postgresql-' + @version if @version.to_f >= 9.4
```

Comment:
What if `@version` is < 9.4? Then `@service` will never get set.

This probably needs to be:

```
@service = 'postgresql'
@service += "-#{@version}" if @version.to_f >= 9.4
```